CSS Folder/AboutMeStyle.css

```diff
@@ -0,0 +1,156 @@
+@font-face {
+    font-family: "Roboto";
+    src: url("../Fonts/Roboto/Roboto-Regular.ttf") format("truetype");
+}
+
+* {
+    font-family: "Roboto", sans-serif;
+}
+
+body {
+    background: radial-gradient(circle, #b597f6 30%, #96c6ea 100%);
+}
+
+main {
+    margin: 0 auto;
+    width: 90%;
+}
+
+a {
+    text-decoration: none;
+    font-weight: bold;
+}
+
+li a {
+    text-decoration: none;
+    font-weight: bold;
+}
+
+li a:hover {
+    color: rgb(214, 210, 238);
+}
+
+nav {
+    background-color: black;
+}
+
+nav ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    margin: 0;
+    padding: 0;
+}
+
+nav ul li {
+    position: relative;
+    flex: 1;
+    text-align: center;
+}
+
+nav ul li:hover {
+    background-color: #152238;
+}
+
+nav ul li a {
+    display: block;
+    color: #fff;
+    padding: 12px 16px;
+    text-decoration: none;
+}
+
+nav ul ul {
+    position: absolute;
+    top: 40px;
+    left: 0;
+    width: 100%;
+    display: none;
+}
+
+nav ul li:hover>ul {
+    display: block;
+}
+
+nav ul ul li {
+    text-align: left;
+    background-color: black;
+    border-bottom: 1px solid black;
+}
+
+nav ul ul li a {
+    padding: 8px 16px;
+}
+
+nav ul ul ul {
+    left: 100%;
+    top: 0;
+}
+
+.aboutmecontainer {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    margin-top: 50px;
+    border: 5px solid white;
+    padding: 25px;
+    align-items: center;
+    justify-content: center;
+}
+
+
+.aboutmecontainer img {
+    float: right;
+    margin-right: 20px;
+    width: 150px;
```

Comment:
avoid width and heights bigger than 80 px to use absolute values, prefer relative units like % vwidth etc. unless it's necessary for the container to wrap correctly